packages/safe-tools-client/app/controllers/schedule.ts

```diff
@@ -14,18 +14,32 @@ export default class Schedule extends Controller {
   @tracked isDepositModalOpen = false;
 
   get safe() {
-    // Should we grab the network nativeToken info here, or expect the sdk method to return it within safeInfo?
-
-    //TODO: get safe info from sdk and format it
+    //TODO: get safe info from sdk and format it,
     return {
       address: '0x8a40AFffb53f4F953a204cAE087219A28771df9d',
-      token: {
-        address: 'eth',
-        name: 'Ethereum',
-        symbol: 'ETH',
-        decimals: 18,
-        balance: 0,
-      },
+      tokens: [
+        {
+          address: 'eth',
+          name: 'Ethereum',
+          symbol: 'ETH',
+          decimals: 18,
+          balance: 1, //in ETH
+        },
+        {
+          address: '0x6B...1d0F', // TODO: check when address is hex, to truncate on template
```

Comment:
Is this the token contract address? I know this is static/placeholder, I’m wondering if this is something we’ll really be supporting 😯

Reply:
Not sure, was just trying to follow a pattern,  I think that it's confusing to show token address and then safe address, maybe we should be showing the nativeToken ? 🤔